pkg/util/util.go

```diff
@@ -77,9 +77,13 @@ type Config struct {
 
 	IsControllerServer bool // if set to true start provisoner server
 	IsNodeServer       bool // if set to true start node server
+
 	// rbd related flag
 	Containerized bool // whether run as containerized
 
+	// cephfs related flags
+	ForceKernelClient bool // force to use the ceph kernel client even if the kernel is < 4.17
```

Comment:
Isnt it better to put the fied as "forcekcephfs" ?

Reply:
Changed it